src/__tests__/plugin.js

```diff
@@ -109,4 +109,30 @@ describe('gatsby-remark-embedder', () => {
 
     expect(remark.stringify(processedAST)).toMatchSnapshot();
   });
+
+  it('can execute custom transformers', async () => {
+    const markdownAST = getMarkdownASTForFile('CustomTransformer');
+
+    const transformer = {
+      shouldTransform: jest.fn(url => url.startsWith('https://domain.com')),
+      getHTML: jest.fn(url => `<iframe href="${url}" />`),
+    };
+
+    const processedAST = await plugin(
+      { cache, markdownAST },
+      { customTransformers: [transformer] }
+    );
+
+    expect(transformer.shouldTransform).toHaveBeenCalledTimes(2);
```

Comment:
Not sure if we need this `expect`s, since the snapshot already proves it won't transform `some-other-domain`, but only `some-domain` 🤔

@kentcdodds What do you think about this?

Reply:
I like the explicitness 👍